service/placeholder.js

```diff
@@ -0,0 +1,56 @@
+const request = require('request');
+const bl = require('bl');
+const _ = require('lodash');
+
+const logger = require( 'pelias-logger' ).get( 'placeholder' );
+
+module.exports = function setup(url) {
+  if (_.isEmpty(url)) {
+    logger.warn('placeholder service disabled');
+
+    return {
+      search: (text, lang, callback) => {
+        callback(`placeholder service disabled`);
+      }
+    };
+
+  }
+
+  logger.info(`using placeholder service at ${url}`);
+  return {
+    search: (text, lang, callback) => {
+      const requestUrl = `${url}/search?text=${text}&lang=${lang}`;
+
+      request
+        .get(requestUrl)
```

Comment:
wouldn't something like this work? https://github.com/pelias/api/blob/master/service/pointinpolygon.js#L12

or does that actually only work if the response is a certain size or smaller?